server/data/activitiesClient.ts

```diff
@@ -3,7 +3,7 @@ import { AuthenticationClient } from '@ministryofjustice/hmpps-auth-clients'
 import config from '../config'
 import logger from '../../logger'
 import { PrisonPayBand, RolloutPrisonPlan } from '../@types/activities'
-import { Context } from '../services/nomisMigrationService'
+import { Context } from '../services/context'
```

Comment:
unfortunately had to move this out of `nomisMigrationService` otherwise we got a circular dependency